pkg/util/time.go

```diff
@@ -2,6 +2,13 @@ package util
 
```

Comment:
Why not move this file to interval and rename the directory to `timeutil`? The `util` directory is difficult to understand for readers.